### PR TITLE
PXB-2677 Document Azure storage option for xbcloud (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
M  source/index.rst
A  source/xbcloud/xbcloud_azure.rst

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud_azure.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
+.. xbcloud_azure:
+
+=============================================================
+Using the xbcloud binary with Microsoft Azure Cloud Storage
+=============================================================
+
+This features is *technical preview* quality.
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud_azure.rst]: This feature is

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud_azure.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+     - AZURE_STORAGE_CLASS
+     - Cloud tier can decrease the local storage required while maintaining the performance. When enabled, this feature has the following categories:
+
+       - Hot - Frequently accessed or modified data
+
+       - Cool - Infrequently accessed or modified data
+
+       - Archive - Rarely accessed or modified data 
+
+The `Azurite open-source emulator <https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio>`__ lets you test Azure applications. The **xbcloud** binary adds the ``--azure-development-storage`` option. This option uses the default ``access_key`` and ``storage account`` of azurite and ``testcontainer`` for the container which can be overwritten with the available options.
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud_azure.rst]: With `Azurite open-source emulator <https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio>`__ you can test Azure applications.

**Review by alina-derkach-oaza — APPROVED** [review]

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud_azure.rst (hunk 2 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud_azure.rst]: Rewritten

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud_azure.rst (hunk 1 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud_azure.rst]: Fixed.

**Review by rahulmalik87 — APPROVED** [review]
LGTM. thank you @patrickbirch

**Review by rahulmalik87 — CHANGES_REQUESTED** [review]
Can you also add details in 
https://www.percona.com/doc/percona-xtrabackup/8.0/xbcloud/xbcloud.html
Supported Cloud Storage Types

**Comment by patrickbirch** [comment]
Added

File: storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud_azure.rst [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+     - AZURE_STORAGE_CLASS
+     - Cloud tier can decrease the local storage required while maintaining the performance. When enabled, this feature has the following categories:
+
+       - Hot - Frequently accessed or modified data
+
+       - Cool - Infrequently accessed or modified data
+
+       - Archive - Rarely accessed or modified data 
+
+With the `Azurite open-source emulator <https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio>`__ you test Azure applications. For testing purposes, the **xbcloud** binary adds the ``--azure-development-storage`` option which uses the default ``access_key`` and ``storage account`` of azurite and ``testcontainer`` for the container. You can be overwrite these options, if needed.
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud_azure.rst]: For testing purposes, the **xbcloud** binary adds the ``--azure-development-storage`` option, which uses the default ``access_key`` and ``storage account`` of azurite and ``testcontainer`` for the container. You can overwrite these options if needed.

Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud_azure.rst]: Thanks.

**Pull request merged**